Header header

uint32 us
uint32 ir1
uint32 ir2
uint32 ir3
uint32 ir4
int32 voltage
